go/client/cmd_simplefs.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"encoding/hex"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+// NewCmdDevice creates the device command, which is just a holder
+// for subcommands.
+func NewCmdSimpleFS(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "simplefs",
+		Usage:        "Perform filesystem operations",
+		ArgumentHelp: "[arguments...]",
+		Subcommands: []cli.Command{
+			NewCmdSimpleFSList(cl, g),
+			NewCmdSimpleFSCopy(cl, g),
+			NewCmdSimpleFSMove(cl, g),
+			NewCmdSimpleFSRead(cl, g),
+			NewCmdSimpleFSRemove(cl, g),
+			NewCmdSimpleFSMkdir(cl, g),
+			NewCmdSimpleFSStat(cl, g),
+			NewCmdSimpleFSGetStatus(cl, g),
+			NewCmdSimpleFSKill(cl, g),
+			NewCmdSimpleFSPs(cl, g),
+			NewCmdSimpleFSWrite(cl, g),
+		},
+	}
+}
+
+func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
+	mountDir := "/keybase"
```

Comment:
Keep in mind that the `Env` isn't configured at this point --- it's often configured *as a result* of `ParseArgv`, from which this function is called.  So I think we need a refactor to just grab raw paths out of the command line in `ParseArgv`, and then to process them during `Run`